rad/prelude/cmd-parsing.rad

```diff
@@ -27,32 +29,57 @@
   (fn [cmd]
     (string-append "Too many arguments for command \"" cmd "\"")))
 
-(def /cmd-1-opts
-  "Matches a command `cmd` with one argument and an optional
-  combination of options that are allowed by opts-map"
-  (fn [cmd opts opts-map]
+(def /cmd-opts
+  "Matches the command `cmd` and a combination of optional parameters, where
+  `opt-match` matches the options to the keys of `multi-ops`. `multi-ops` is a
+  dict of option-keys with multiple arguments. If no options are passed, the
+  `default` is returned."
+  (fn [cmd opts multi-opts opt-match default help]
+    (def-rec parse-cli
+      (fn [args default]
```

Comment:
You don’t need to pass `default`. The value is always available in the closure from the arguments to `/cmd-opts`.